doc/tutorials/upgrading_configuration.rst

```diff
@@ -0,0 +1,70 @@
+.. _upgrading_configuration:
+
+Upgrading ACRN Configurations to Recent Releases
+################################################
+
+The configurations files, introduced in :ref:`acrn_config_data`, are refined
+every release for richer features, clearer organizations and more user friendly
+representations. Due to the strict validation ACRN adopts, configuration files
+for a former release may not work for a latter if they are not upgraded. This
+tutorial introduces the steps to upgrade those files.
+
+Board XML
+*********
+
+A board XML file lists the characteristics of a board and is generated by the
+board inspector. Each release of ACRN adds some more information into that file
+in order to enable new features. Thus, always regenerate the board XML using the
+board inspector of the new release when you upgrade.
+
+For the steps to use the latest board inspector, refer to
+:ref:`board_inspector_tool`.
+
+Scenario XML
+************
+
+A scenario XML file captures the configurations of hypervisor features and
+definitions of VMs. Starting from v3.0, the highly recommended way to upgrade a
+scenario XML is to use the scenario XML upgrader which can be found at
+`misc/config_tools/scenario_config/upgrader.py`. Invoke this script from the
+command line in the following way:
+
+   .. code-block:: bash
+
+      misc/config_tools/scenario_config/upgrader.py <your scenario XML> <out file>
+
+`<your scenario XML>` is the path to the scenario XML file for a previous
+release. This script will generate a new scenario XML file, located at `<out
+file>`, by migrating, following the latest schema, as many data in `<your
```

Comment:
Can this be a filename *and* a path (relative or absolute)?

Reply:
It can be an arbitrary path (relative to CWD or absolute) which will be interpreted as the path of the generated file, assuming the parent directory of that path exists.